common/djangoapps/static_replace/__init__.py

```diff
@@ -109,6 +110,13 @@ def wrap_part_extraction(match):
         prefix = match.group('prefix')
         quote = match.group('quote')
         rest = match.group('rest')
+
+        # Don't rewrite XBlock resource links.  Probably wasn't a good idea that /static
+        # works for actual static assets and for magical course asset URLs....
+        full_url = prefix + rest
+        if full_url.startswith(XBLOCK_STATIC_RESOURCE_PREFIX):
+            return original
```

Comment:
We return `original` here because the replacement function is expected to return the modified URL with double quotes included.